### PR TITLE
Interactive dictionary

### PR DESCRIPTION
**Pull request by Tomash996**
Interactive dictionary

### CODE REVIEW

File: src/Dictionary.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import java.util.HashMap;
+
+public class Dictionary {
+    private HashMap<String, String> dict = new HashMap<>();
```
Comment (vasylboliuk) [inline src/Dictionary.java]: Add white space
Also, init map, you can use just Map<String,String>

File: src/Dictionary.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    public String look(String key) {
+        if (dict.containsKey(key)) return dict.get(key);
+        return "Cant find entry for " + key; //returns nothing value
+    }
+}
+
+class Main {
+    public static void main(String[] args) {
```
Comment (vasylboliuk) [inline src/Dictionary.java]: add white space here

File: src/Dictionary.java (hunk 1 above)
Comment (Tomash996) [reply src/Dictionary.java]: resolved

File: src/Dictionary.java (hunk 2 above)
Comment (Tomash996) [reply src/Dictionary.java]: resolved

**Pull request merged**